db/migrate/20170207192912_encrypt_otp_secret.rb

```diff
@@ -0,0 +1,22 @@
+class EncryptOtpSecret < ActiveRecord::Migration
+  class User < ActiveRecord::Base
+  end
+
+  def up
+    encrypt_otp_secret_key
+  end
+
+  def down
+  end
+
+  def encrypt_otp_secret_key
+    User.where.not(otp_secret_key: nil).find_in_batches.with_index do |users, batch|
+      users.each do |user|
+        encrypted_attribute = EncryptedAttribute.new_from_decrypted(user.otp_secret_key)
```

Comment:
The problems we had in the past were having old migrations that referenced code that was no longer present. This is not the case here. This migration is using existing code with existing columns. It is merely populating an existing column directly via Postgres.